services/buildbot/master/master.cfg

```diff
@@ -6,17 +6,6 @@
 # <http://buildbot.twistedmatrix.com/>. Passwords and other secret
 # information are loaded from a neighboring file called 'private.py'.
 
-# Enable TCP keepalives.  The build master is terrible at noticing
```

Comment:
This is used by the buildslaves; the workers and master still communicate over PB.